pages/experiments.js

```diff
@@ -40,32 +92,37 @@ export const getStaticProps = async ({ locale }) => {
     {
       id: 1,
       title: "Lorem Ipsum",
```

Comment:
Could you just add numbers to these maybe? Otherwise, you can't really see if they change because they all seem to say the same thing.

Reply:
Sure thing